libpimeval/src/pimParamsDDRDram.cpp

```diff
@@ -0,0 +1,154 @@
+    m_tRFC2 = std::stoi(pimUtils::getParam(params, "tRFC2"));
+    m_tRFC4 = std::stoi(pimUtils::getParam(params, "tRFC4"));
+    m_tREFI = std::stoi(pimUtils::getParam(params, "tREFI"));
+    m_tRPRE = std::stoi(pimUtils::getParam(params, "tRPRE"));
+    m_tWPRE = std::stoi(pimUtils::getParam(params, "tWPRE"));
+    m_tRRD_S = std::stoi(pimUtils::getParam(params, "tRRD_S"));
+    m_tRRD_L = std::stoi(pimUtils::getParam(params, "tRRD_L"));
+    m_tWTR_S = std::stoi(pimUtils::getParam(params, "tWTR_S"));
+    m_tWTR_L = std::stoi(pimUtils::getParam(params, "tWTR_L"));
+    m_tFAW = std::stoi(pimUtils::getParam(params, "tFAW"));
+    m_tWR = std::stoi(pimUtils::getParam(params, "tWR"));
+    m_tWR2 = std::stoi(pimUtils::getParam(params, "tWR2"));
+    m_tRTP = std::stoi(pimUtils::getParam(params, "tRTP"));
+    m_tCCD_S = std::stoi(pimUtils::getParam(params, "tCCD_S"));
+    m_tCCD_L = std::stoi(pimUtils::getParam(params, "tCCD_L"));
+    m_tCKE = std::stoi(pimUtils::getParam(params, "tCKE"));
+    m_tCKESR = std::stoi(pimUtils::getParam(params, "tCKESR"));
+    m_tXS = std::stoi(pimUtils::getParam(params, "tXS"));
+    m_tXP = std::stoi(pimUtils::getParam(params, "tXP"));
+    m_tRTRS = std::stoi(pimUtils::getParam(params, "tRTRS"));
+
+    m_VDD = std::stod(pimUtils::getParam(params, "VDD"));
+    m_IDD0 = std::stoi(pimUtils::getParam(params, "IDD0"));
+    m_IPP0 = std::stod(pimUtils::getParam(params, "IPP0"));
+    m_IDD2P = std::stoi(pimUtils::getParam(params, "IDD2P"));
+    m_IDD2N = std::stoi(pimUtils::getParam(params, "IDD2N"));
+    m_IDD3P = std::stoi(pimUtils::getParam(params, "IDD3P"));
+    m_IDD3N = std::stoi(pimUtils::getParam(params, "IDD3N"));
+    m_IDD4W = std::stoi(pimUtils::getParam(params, "IDD4W"));
+    m_IDD4R = std::stoi(pimUtils::getParam(params, "IDD4R"));
+    m_IDD5AB = std::stoi(pimUtils::getParam(params, "IDD5AB"));
+    m_IDD6x = std::stoi(pimUtils::getParam(params, "IDD6x"));
+
+    m_channelSize = std::stoi(pimUtils::getParam(params, "channel_size"));
+    m_channels = std::stoi(pimUtils::getParam(params, "channels"));
+    m_busWidth = std::stoi(pimUtils::getParam(params, "bus_width"));
+    m_addressMapping = pimUtils::getParam(params, "address_mapping");
+    m_queueStructure = pimUtils::getParam(params, "queue_structure");
+    m_refreshPolicy = pimUtils::getParam(params, "refresh_policy");
+    m_rowBufPolicy = pimUtils::getParam(params, "row_buf_policy");
+    m_cmdQueueSize = std::stoi(pimUtils::getParam(params, "cmd_queue_size"));
+    m_transQueueSize = std::stoi(pimUtils::getParam(params, "trans_queue_size"));
+
+    m_epochPeriod = std::stoi(pimUtils::getParam(params, "epoch_period"));
+    m_outputLevel = std::stoi(pimUtils::getParam(params, "output_level"));
+  } catch (const std::invalid_argument& e) {
+    std::string errorMessage("PIM-Error: Missing or invalid parameter: ");
+    errorMessage += e.what();
+    errorMessage += "\n";
+    throw std::invalid_argument(errorMessage);
+  }
+}
```

Comment:
There are warning signs for missing line break at the end of a few files

Reply:
Thanks! Updated!